types/metered_message.go

```diff
@@ -21,16 +21,7 @@ type MeteredMessage struct {
 	Message  `json:"message"`
 	GasPrice AttoFIL  `json:"gasPrice"`
 	GasLimit GasUnits `json:"gasLimit"`
-}
-
-// Unmarshal a message from the given bytes.
-func (msg *MeteredMessage) Unmarshal(b []byte) error {
-	return cbor.DecodeInto(b, msg)
-}
-
-// Marshal the message into bytes.
-func (msg *MeteredMessage) Marshal() ([]byte, error) {
-	return cbor.DumpObject(msg)
+	// Pay attention to Equals() if updating this struct.
```

Comment:
we could enforce this with a unit test that uses reflection to assert the fields in message are what equal tests against

Reply:
+1 this has saved me a few times with block: https://github.com/filecoin-project/go-filecoin/blob/master/types/block_test.go#L107

Reply:
Great suggestion, thanks